txserver/handler.go

```diff
@@ -100,8 +95,8 @@ func cancel_set_sell(ctx *context.Context, command *Command) ([]byte, error) {
 	return []byte{}, nil
 }
 
-func dumplog(ctx *context.Context, command *Command) ([]byte,error) {
-	eventCollection := mongoClient.Database("test").Collection("Events")
+func dumplog(ctx *context.Context, command *Command) ([]byte, error) {
+	eventCollection := client.Database("test").Collection("Events")
```

Comment:
minor: we should rename this as mongoClient. Otherwise, 'client' can mean any client